src/sentry/templates/sentry/integrations/gitlab-config.html

```diff
@@ -3,28 +3,69 @@
 {% load sentry_assets %}
 {% load i18n %}
 
+{% block css %}
+  <link href="{% asset_url "sentry" "dist/sentry.css" %}" rel="stylesheet"/>
+  <style>
+    ol li {
+      padding:10px 0px;
+    }
+    ul {
+      padding: 10px;
+      background-color: #f7f8f9;
+      margin-top: 10px;
+      border-radius: 3px;
+    }
+    ul li {
+      list-style-type: none;
+      font-size: 90%;
+      font-family: Monaco, Consolas, 'Courier New', monospace;
+      padding:0;
+    }
+    a.btn.btn-primary{
+      margin-bottom: 30px;
+    }
+  </style>
+{% endblock %}
+
 {% block wrapperclass %} narrow auth {% endblock %}
 {% block modal_header_signout %} {% endblock %}
 
 {% block title %} {% trans "GitLab Setup" %} | {{ block.super }} {% endblock %}
 
 {% block main %}
 <h3>GitLab Configuration</h3>
+{% if form %}
+  <form action="" method="post" class="form-stacked">
+    {% csrf_token %}
+    <input type="hidden" name="provider" value="gitlab" />
 
-<form action="" method="post" class="form-stacked">
-  {% csrf_token %}
-  <input type="hidden" name="provider" value="gitlab" />
-
-  <p>{% trans "Configure GitLab to use with Sentry." %}</p>
+    <p>{% trans "Configure GitLab to use with Sentry." %}</p>
 
-  {{ form|as_crispy_errors }}
+    {{ form|as_crispy_errors }}
 
-  {% for field in form %}
-   {{ field|as_crispy_field }}
-  {% endfor %}
+    {% for field in form %}
+    {{ field|as_crispy_field }}
+    {% endfor %}
 
-  <fieldset class="form-actions">
-    <button type="submit" class="btn btn-primary" name="save_mappings" value="url">{% trans "Submit" %}</button>
-  </fieldset>
-</form>
+    <fieldset class="form-actions">
+      <button type="submit" class="btn btn-primary" name="save_mappings" value="url">{% trans "Submit" %}</button>
+    </fieldset>
+  </form>
+{% else %}
```

Comment:
I thought about creating another template, but this seemed minor enough that using the same one was ok.

Reply:
It is a trade-off. I would lean towards having a separate template so that changes to one have no risk of breaking the other. This uses fewer lines of code, it is a trade-off :smile: